DfE.FindInformationAcademiesTrusts/Pages/StringFormatConstants.cs

```diff
@@ -2,6 +2,6 @@ namespace DfE.FindInformationAcademiesTrusts.Pages;
 
 public static class StringFormatConstants
 {
-    public const string ViewDate = "d MMM yyyy";
-    public const string SortDate = "yyyyMMdd";
+    public const string FullDateFormat = "dd MMM yyyy";
```

Comment:
I disagree with the wording here, its not a full date format since month is shortened. I think we should use display date format for this
```suggestion
    public const string DisplayDateFormat = "dd MMM yyyy";
```